conf/github.yaml.template

```diff
@@ -0,0 +1,13 @@
+GITHUB:
+  FOREMAN:
+    ORG: theforeman
+    REPO: foreman
+    RULES:
+     - PATH:
+       MARKER:
```

Comment:
I might think that we can do one more layer deeper at file level, this needs to be check with Dev as well.